src/state/reducer.ts

```diff
@@ -35,10 +38,8 @@ const reducerMap = {
   ERROR: onError
```

Comment:
As we're removing the `onComplete` function, these two lines can also be removed
```
ON_REDIRECT_COMPLETE: onComplete,
GET_ACCESS_TOKEN_COMPLETE: onComplete,
```